releasenotes/notes/fleet-setup-keep-yaml-comments-40d804ecd60e916c.yaml

```diff
@@ -0,0 +1,6 @@
+---
+enhancements:
+  - |
+    The Datadog Installer introduced in 7.65.0 now keeps YAML comments
+    when updating the Agent configuration during installation.
+    Some formatting changes can be expected.
```

Comment:
We should either remove this line or better explain what we mean. This wording is not very explicit for users.

Reply:
maybe something like?
```
While comment text is preserved, its format may not be. Changes to whitespace, indentation, or blank lines may occur.
```